### ISSUE TITLE
Privileges refactor

### ISSUE THREAD

**Issue by pitaj** [NodeBB/NodeBB#5736]
- [ ] Create a centralized privileges interface (by group / by user instead of by category)
- [ ] Split up moderation permissions
- [ ] Add global privileges
- [ ] Change global moderators to a default group with the global moderation permission
- [ ] ? switch to not using groups for storing permissions ?

This would cover #5569, some of #3914, and could cover #4652 and #5127 as well.

**Comment by ELadner** [comment]
Could it be possible with this scheme to create a category where users can't see other users topics but can create a topic and always view their own topic?

Like a support forum where users may have sensitive issues.  Other users don't need to see the topic, but the user that creates it can interact with the support team that can read the whole category.

**Comment by julianlam** [comment]
No, that is too obscure a use case to condition for, _I think_. Traditionally we've opted to do this via third party plugins.

nodebb-plugin-support-forum being the one you're looking for.

**Comment by dangerusslee** [comment]
I would be interested in the ability to give groups certain admin privileges - like managing categories for instance.

**Comment by aStonedPenguin** [comment]
@julianlam No notable forum software doesn't allow you to give groups fine tuned permissions. Decisions like that seriously hold NodeBB back from widespread use.

**Comment by mohammadhassan99** [comment]
Up 😊

**Comment by exomarty** [comment]
Up

**Comment by julianlam** [comment]
For what it's worth, a number of these changes have been implemented, with global privileges and purge now being a separate privilege no longer bundled with moderation.

**Comment by gaellelo** [comment]
@julianlam does it mean that global moderators shouldn't be able to purge anymore? 
If so how cause under our settings, if they have moderation rights they can still purge for example.

**Comment by julianlam** [comment]
No, global moderators should still be able to purge.

**Comment by Fudster** [comment]
Bump? Would be nice to break these permissions down more because I want my mods to be able to delete posts but not edit

**Comment by mySebbe** [comment]
The checklist is now partly stale: Core has separate post edit/history/delete, topic delete, purge, and delegated admin privileges, and `develop` adds further post-edit/all-category work. The remaining items mix two different projects — deny precedence and splitting individual moderator actions such as pin/lock/move. Could #5736 remain a tracker while each remaining permission gets its own contract and test matrix? That would let small authorization changes land without coupling them to a system-wide deny hierarchy.
